Dockerfile.web

```diff
@@ -1,25 +1,9 @@
 FROM ocaml/opam:debian-12-ocaml-4.14@sha256:06d58a5cb1ab7875e8d94848102be43f6492e95320e8e9a9ecb9167654d0ee3f AS build
-RUN sudo ln -f /usr/bin/opam-2.1 /usr/bin/opam && opam init --reinit -ni
+RUN sudo ln -f /usr/bin/opam-2.2 /usr/bin/opam && opam init --reinit -ni
```

Comment:
I know it's a small change, but perhaps we should save this update for its own PR, so it'll be easier to isolate the cause if anything breaks between this update and the change in dependency management. WDYT, am I being paranoid?

Reply:
I'm happy to leave this change until another time as it is unrelated to the other changes.